src/main/index.ts

```diff
@@ -62,9 +62,11 @@ const di = getDi();
 
 app.setName(appName);
 
-di.runSetups().then(() => {
-  injectSystemCAs();
+app.on("ready", async () => {
```

Comment:
App start order was previously wrong... any async setups caused `app.on("ready")` registration happen too late -> we missed it completely.

Reply:
So that we can catch these async errors I would suggest a slightly different shape:

```
async function main() {
  const di = getDi();

  await app.whenReady();
  await di.runSetups();

  ...
}

main().catch(error => {
  console.error(error);
  dialog.showErrorBox("Lens Startup Failed", String(error));
  app.exit(1);
});
```